packages/timeline-state-resolver/src/integrations/casparCG/index.ts

```diff
@@ -111,42 +96,54 @@ export class CasparCGDevice extends DeviceWithState<State, DeviceOptionsCasparCG
 	 */
 	async init(initOptions: CasparCGOptions): Promise<boolean> {
 		this.initOptions = initOptions
-		this._useScheduling = initOptions.useScheduling
-		this._ccg = new CasparCG({
+		this._ccg = new BasicCasparCGAPI({
 			host: initOptions.host,
 			port: initOptions.port,
-			autoConnect: true,
-			virginServerCheck: true,
-			onConnectionChanged: (connected: boolean) => {
-				this._connected = connected
-				this._connectionChanged()
-			},
 		})
 
-		this._ccg.on(CasparCGSocketStatusEvent.CONNECTED, (event: CasparCGSocketStatusEvent) => {
+		this._ccg.on('connect', () => {
 			this.makeReady(false) // always make sure timecode is correct, setting it can never do bad
 				.catch((e) => this.emit('error', 'casparCG.makeReady', e))
-			if (event.valueOf().virginServer === true) {
-				// a "virgin server" was just restarted (so it is cleared & black).
-				// Otherwise it was probably just a loss of connection
 
-				this._ccgState.softClearState()
-				this.clearStates()
-				this.emit('resetResolver')
-			}
+			this._connected = true
+			this._connectionChanged()
+
+			// TODO - maybe add this back based on info command
+			// if (event.valueOf().virginServer === true) {
```

Comment:
Delete commented code

Reply:
this will be gone in the next PR where I combined some of later fixes from this integration